packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -966,11 +950,7 @@ export class AppMainContainer extends Component<
                   )}
                 </span>
               }
-              tooltip={
-                isDisconnected && !isAuthFailed
-                  ? 'Server disconnected'
-                  : 'User Settings'
-              }
+              tooltip="Server disconnected"
```

Comment:
This should still be `User Settings` if we're not disconnected.

That said, I'm not even sure it's possible to see/use this button in the disconnected state any more since we're showing a full screen loader

Reply:
Yeah, can't see this button in the disconnected state, so removing.